### PR TITLE
updated Stencil path

### PR DESCRIPTION
**Pull request by ilyapuchka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 [![Platform](https://img.shields.io/cocoapods/p/StencilSwiftKit.svg?style=flat)](http://cocoadocs.org/docsets/StencilSwiftKit)
 ![Swift 4.0](https://img.shields.io/badge/Swift-4.0-orange.svg)
 
-`StencilSwiftKit` is a framework bringing additional [Stencil](https://github.com/kylef/Stencil) nodes & filters dedicated to Swift code generation.
+`StencilSwiftKit` is a framework bringing additional [Stencil](https://github.com/stencilproject/Stencil) nodes & filters dedicated to Swift code generation.
```
Comment (AliSoftware) [inline README.md]: I think there are more in the README? (Lines 52 and 61 especially)

**Comment by ilyapuchka** [comment]
@AliSoftware I can't merge this as I'm not "collaborator" of this repo

**Comment by AliSoftware** [comment]
I just invited you

File: StencilSwiftKit.podspec [hunk 2]
```diff
@@ -5,7 +5,7 @@ Pod::Spec.new do |s|
 
   s.description  = <<-DESC
                    This pod contains some additional nodes and filters for
-                   [Stencil](https://github.com/kylef/Stencil).
+                   [Stencil](https://github.com/stenciliproject/Stencil).
```
Comment (djbe) [inline StencilSwiftKit.podspec]: Typo.

**Comment by ilyapuchka** [comment]
@AliSoftware still didn't get the invite, can you just merge it? @djbe I've addressed your comment, good catch.

**Comment by djbe** [comment]
Huh, weird. I wanted you to have the honors 😄

**Comment by AliSoftware** [comment]
Ah, I see now, I invited you as Collaborator, with write access, but the master branch was still only allowing commits from the CoreTeam, not from Collaborators. I've added you to the list of authorized people to commit on master, should be good for the next PR 😅

**Pull request merged**